utilities/cli/holohub.py

```diff
@@ -914,10 +953,19 @@ def handle_test(self, args: argparse.Namespace) -> None:
         if args.verbose:
             ctest_cmd += "-VV "
 
+        # If coverage is requested, ensure gcov is available; install gcc if missing (as root)
+        if getattr(args, "coverage", False):
+            preinstall_cmd = (
+                "if ! command -v gcov >/dev/null 2>&1; then "
+                "(apt-get update && apt-get install -y gcc) || true; fi; "
```

Comment:
I think this installation commands should be merged into the new `utilities/setup/coverage.sh`?

Reply:
Good catch - let me remove

Reply:
I see the inline `apt-get install -y gcc` is removed in the latest version, can we add gcc installation in `utilities/setup/coverage.sh`?